mlflow/java/spark/pom.xml

```diff
@@ -29,17 +44,32 @@
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-sql_${scala.compat.version}</artifactId>
-      <version>3.0.0-preview</version>
+      <version>3.5.0</version>
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_${scala.compat.version}</artifactId>
-      <version>3.0.0-preview</version>
+      <version>3.5.0</version>
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-catalyst_${scala.compat.version}</artifactId>
-      <version>3.0.0-preview</version>
+      <version>3.5.0</version>
+    </dependency>
+    <dependency>
```

Comment:
why do we add these dependencies? It seems only mlflow/java/scoring are using them

Reply:
mlflow/java/spark uses them: https://github.com/mlflow/mlflow/blob/master/mlflow/java/spark/src/main/scala/org/mlflow/spark/autologging/ReplAwareSparkDataSourceListener.scala#L3-L6